### ISSUE TITLE
posthtml-hbs (?)

### ISSUE THREAD

**Issue by stevenvachon** [posthtml/posthtml#193]
The link in the readme is dead.

**Comment by michael-ciniawsky** [comment]
@stevenvachon Yep, when I updated the docs I was sure to release that shortly after, was obviously wrong after all :) and got distracted with other stuff. Would you use it, I can quikly do it, but if not tbh I discard it and remove the link form the README when I update examples for grunt, gulp, webpack

**Comment by stevenvachon** [comment]
I have a need for running handlebars before posthtml, which could benefit from a plugin. The only benefit, I think, would be a persistent API.

**Comment by michael-ciniawsky** [comment]
> I have a need for running handlebars before posthtml

Alright I will make it

> The only benefit, I think, would be a persistent API

Yep exactly, the goal is to have a convenience wrapper for `.hbs` files, helping folks to migrate/use PostHTML with Handlebars. If you have anything in mind please leave a comment here :).

```js
const hbs = require('posthmtl-hbs')({ locals*: {} , partials: '', helpers: '' })
// *locals === data/context

posthtml(plugins).process(file.hbs, { parser: hbs })...
```

Especially how the 'API' to register partials/helpers should look like

**Comment by stevenvachon** [comment]
@michael-ciniawsky or we could write a "posthtml-template" plugin that uses [consolidate](https://npmjs.com/consolidate), as to avoid writing countless specific plugins. Such could replace your [posthtml-jade](https://npmjs.com/posthtml-jade).

**Comment by stevenvachon** [comment]
@michael-ciniawsky thoughts?

**Comment by selfagency** [comment]
this ever happen?

**Comment by GitScrum** [comment]
@stevenvachon I'm afraid not (maybe you have ideas or want to do?
